tools/osbuilder/rootfs-builder/nvidia/nvidia_rootfs.sh

```diff
@@ -45,6 +46,7 @@ setup_nvidia-nvrc() {
 
 setup_nvidia-gpu-admin-tools() {
 	local TARGET="nvidia-gpu-admin-tools"
+	local TARGET_VERSION="v2024.12.06"
```

Comment:
Are these versions anticipated to change often? Should we consider moving them to versions.yaml?

Reply:
No they're not meant to change often. For now keep them in the rootfs.sh.